Monika After Story/game/script-topics.rpy

```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
+                                                    m "{i}Ehehe{/i}~"
+                                        "Those were just lucky guesses.":
+                                            m "Well, I guess you managed to fool me [player]~"
+                                            m "So that means you haven't watched it yet huh."
+#This is where I need the code or a way to somehow jump to another part of this topic#
+                                "Neiman":
+                                    m "Wrong! Caught you [player]!"
+                                    m "Ahaha!"
+#This is where I need the code or a way to somehow jump to another part of this topic# 
+                        "Nicole":
+                            m "Wrong! Caught you [player]!"
+                            m "Ahaha!"
+#This is where I need the code or a way to somehow jump to another part of this topic#
+                        "Terrance":
+                            m "Wrong! Caught you [player]!"
+                            m "Ahaha!"
+#This is where I need the code or a way to somehow jump to another part of this topic#
+                "I was just kidding."
+        "No.":
+            m "It's a pretty good movie."
+            m "Not my favourite, but if was nice."
+#I need here way to code if the player has seen the jazz topic and said they like jazz. If it's impossible, then nevermind#
+#It would go something like:#
+# m "You once said you like jazz right? So I think you might like this one."#
+            m "You want me to give you a short synopsis [player]?"
+            menu:
+                "Sure.":
+                    m "Algirht!"
+                    m "So it's abou-"
+                    m "..."
+                    m "I might get into some spoilers, would you mind that [player]?"
+                    menu:
+                        "Not really.":
+                            m "Great!"
+                            m "So this movie is about a guy named Andrew who wanted to become one of the best jazz drummers in the world."
+                            m "His teacher, Fletcher, is one of the best jazz instructiors there."
+                            m "He brought Andrew to be one of his drummers for his band."
+                            m "But then you find out that he's a really abusive teacher."
+                            m "He'd belittle you, berate you, physically and emotionally abuse you."
+                            m "And even hurl a chair!"
+                            m "I'm not kidding!"
+                            m "The whole movie is about the relationship between Andrew handling Fletcher's '{i}questionable{/i}' teaching methods."
+                            m "And what an ambitious student is willing to go through for his dream."
+                            m "And how far a teacher is willing to go to push his students to reach greatness."
+                        "I would mind."
```

Comment:
missing `:` at the end of `"I would mind"`

Reply:
done